pkg/cmd/config.go

```diff
@@ -519,6 +519,11 @@ features:
       # be used.
       caFile: ""
 
+systemPackages:
+  # if set (true by default), will configure OS package manager with
```

Comment:
I'd maybe say `will add Docker and Kubernetes repositories to OS package manager` instead.